android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDatabaseMigration.java

```diff
@@ -6,18 +6,19 @@
 
 public class RealmDatabaseMigration implements RealmMigration {
 
-    public static long DB_VERSION = 1;
+    public static long DB_VERSION = 2;
 
     @Override
     public void migrate(DynamicRealm realm, long oldVersion, long newVersion) {
         // DynamicRealm exposes an editable schema
         RealmSchema schema = realm.getSchema();
-        if(oldVersion == 0) {
-            schema.get("Session")
+        if(oldVersion == 1) {
+            schema.get("Event")
                     .addField("created-at", String.class)
                     .addField("deleted-at", String.class)
-                    .addField("submitted-at", String.class)
-                    .addField("is-mail-sent", Boolean.class);
+                    .addField("original-image-url", String.class)
+                    .addField("large-image-url", String.class)
+                    .addField("icon-image-url", String.class);
```

Comment:
And if renaming fields, migration is not needed. Just don't rename the fields in model, only the jsonproperty and getters and setters